tests/README.md

```diff
@@ -14,29 +14,34 @@ When writing new tests it is useful to run selected tests locally and isolated.
 ```
 docker build -f docker/actinia-core-tests/Dockerfile -t actinia-test .
 ```
-3. Create a local `tmp_tests` folder and fill it with the files:
+3. To run only a few tests you can mark the tests for development with
+`@pytest.mark.dev` and add `import pytest` to the file.
 
-- `tests/__init__.py`
-- `tests/conftest.py`
-- `tests/test_resource_base.py`
-- your_new_test(s).py
-
-
-4. Start the docker container and mount your local `tmp_tests` folder:
+4. Start the docker container and mount your `tests` folder:
 
 ```
-docker run -it -v /path/to/tmp_tests:/src/actinia_core/tests actinia-test:latest -i
+docker run -it -v /path/to/tests:/src/actinia_core/tests actinia-test:latest -i
 
-# If you want to run all tests from actinia_core/tests instead of selected ones you can use the following command:
+# If you are not developing the tests you can run tests using the following command:
 docker run -it actinia-test:latest -i
 
 ```
 
 5. To execute the test(s) run:
 
 ```
+# execute all tests
 make test
 
+# execute only integration tests (not unittests)
+make notunittest
+
+# execute only unit tests
+make unittest
+
+# execute tests marked with '@pytest.mark.dev'
+make devtest
+
```

Comment:
Not sure where to put it, but somewhere in the test-README I think the new ` tests/unittests/test_version.py` should be mentioned as an example for pytest-decorator usage, because I find this `pytest.mark.parametrize`-thing very helpful.  Something like:
```
For best practice examples on the use of pytest-decorators, see `unittests/test_version.py`
```